packages/wrangler/src/type-generation.ts

```diff
@@ -144,7 +144,7 @@ export function constructType(
 		if (useRawVal) {
 			return `${typeKey}: ${value};`;
 		}
-		return `${typeKey}: "${value.replace(/"/g, '\\"')}";`;
+		return `${typeKey}: "${value.replace(/"/g, '\\"').replace(/\n/g, '\\n')}";`;
```

Comment:
I wonder if we'd be able to get away with something simpler here—maybe `JSON.stringify()` of the vlaue?

Reply:
Thank you for the suggestion! I've updated the constructType function to use JSON.stringify() for escaping string values. Please review the changes and let me know if there are any further improvements needed.